### ISSUE TITLE
3.4.1 Merge Checklist

### ISSUE THREAD

**Issue by dtrudg** [apptainer/singularity#4450]
PRs that need to be merged into the release-3.4 branch:

- [x] #4333 Fix fakeroot build with zypper bootstrap (via #4456)
- [x] #4341 Fix mconfig seccomp detection on Opensuse (via #4458)
- [x] #4370 Fix compiles on non 64-bit architectures (via #4457)
- [x] #4380 Fix FuseMount check causing to always set PID namespace (via #4459)
- [x] #4321 Add go-generate script (via #4462)
- [x] #4375 small bugfix: Fix arguments order (via #4462)
- [x] #4413 Pass Mac hostname information to VM to be used (via #4460)
- [x] #4339 Fix fakeroot build with pacstrap and debootstrap (via #4465)
- [x] #4393 Update to scs-library-client 0.4.4. Fix Library V2 uses large amounts of RAM, can fail
             (via #4466) 
- [x] #4449 Fix ioctl errno handling for LOOP_SET_STATUS64 (via #4461)

In addition, to unblock work here due to test failures, we need:

- [x] #4468 Stop key tests failing

**Comment by ArangoGutierrez** [comment]
#4333 merging via https://github.com/sylabs/singularity/pull/4456

**Comment by dtrudg** [comment]
#4370 merging via #4457

**Comment by ikaneshiro** [comment]
#4341 merging via #4458

**Comment by dtrudg** [comment]
#4380 merging via #4459

**Comment by ArangoGutierrez** [comment]
#4413 merged via  https://github.com/sylabs/singularity/pull/4460

**Comment by ikaneshiro** [comment]
#4321 and #4375 merging via #4462

**Comment by dtrudg** [comment]
#4449 merging via 4461

**Comment by dtrudg** [comment]
Closing this issue now. These have all been merged, and I've done a pass of manual tests over the state of the `release-3.4` branch at 5d9975e95fa79f2341526477e2b39d2fdfbd2921 to confirm
